R/Features-processing.R

```diff
@@ -148,7 +148,7 @@ setMethod("scaleTransform", "Features",
                   stop("Provide index or name of assay to be processed")
               if (length(i) != 1)
                   stop("Only one assay to be processed at a time")
-              if(is.numeric(i)) i <- names(object)[[i]]
+              if (is.numeric(i)) i <- names(object)[[i]]
               addAssay(object,
                        scaleTransform(object[[i]], center, scale),
                        name,
```

Comment:
I confused below with the 
```r
assayLinks = .createAssayLinkOneToOne(seFrom = object[[i]],
                                      seTo = object[[i]],
                                      nameFrom = i,
                                      nameTo = name))
```
1. Why call the (internal) `.createAssayLinkOneToOne` function, and not `createAssayLinkOneToOne`?
2. The `seFrom` and `seTo` are the same?

Reply:
The also applies to the other similar processing functions.

Reply:
Depending on these comments, I will need to dig deeper in the unit tests and possibly in the validity method.